nixos/modules/services/networking/rosenpass.nix

```
@@ -0,0 +1,178 @@
+{
+  config,
+  lib,
+  options,
+  pkgs,
+  ...
+}:
+with builtins;
+with lib; let
+  cfg = config.services.rosenpass;
+  opt = options.services.rosenpass;
+in {
+  options.services.rosenpass = with types; {
+    enable = mkEnableOption "Whether to enable the Rosenpass service to provide post-quantum secure key exchange for WireGuard.";
```

`mkEnableOption` takes a `name` parameter, not a full option description.

```suggestion
    enable = mkEnableOption "Rosenpass";
```